README.md

```diff
@@ -16,17 +16,15 @@ class ViewController: UIViewController {
         Navigate.shared.configure(controller: navigationController)
             .set(title: "Hello SwiftUIKit")
             .setRight(barButton: BarButton {
-                Button({
+                Button("Button 0") {
```

Comment:
in the screen shot you may wanna make it look like a nav button instead of plain black

Reply:
Huh... I see blue on my side

https://github.com/0xLeif/SwiftUIKit/tree/_develop

Looks like it just shows the binary file was modified and not images being replaced. Same name... :/
![CleanShot 2020-07-06 at 17 13 01](https://user-images.githubusercontent.com/8268288/86655082-0fe27180-bfac-11ea-9238-b2d46ec0b914.png)